### PR TITLE
speed up 2nd try of ITs by not recompiling frontend twice

### PR DESCRIPTION
**Pull request by manolo**

### CODE REVIEW

File: vaadin-list-box-flow-parent/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java [hunk 1]
```diff
@@ -32,7 +32,7 @@
  *            type of the item represented by this component
  */
 @Tag("vaadin-item")
-@NpmPackage(value = "@vaadin/vaadin-item", version = "2.3.0")
+@NpmPackage(value = "@vaadin/vaadin-item", version = "2.3.0-beta1")
```
Comment (tulioag) [inline vaadin-list-box-flow-parent/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java:35]: Is this right?

File: vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java [hunk 2]
```diff
@@ -135,7 +135,7 @@
  *
  */
 @Tag("vaadin-grid")
-@NpmPackage(value = "@vaadin/vaadin-grid", version = "5.7.1")
+@NpmPackage(value = "@vaadin/vaadin-grid", version = "5.7.0")
```
Comment (tulioag) [inline vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:138]: Is this right?

**Review by tulioag — CHANGES_REQUESTED** [review]

File: vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java (hunk 2 above)
Comment (manolo) [reply vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java:138]: It's currently in origin master https://github.com/vaadin/vaadin-grid-flow/blob/master/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/Grid.java#L138

File: vaadin-list-box-flow-parent/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java (hunk 1 above)
Comment (manolo) [reply vaadin-list-box-flow-parent/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java:35]: Currently in master https://github.com/vaadin/vaadin-list-box-flow/blob/master/vaadin-list-box-flow/src/main/java/com/vaadin/flow/component/listbox/VaadinItem.java#L35

**Review by tulioag — APPROVED** [review]

**Pull request merged**